hnn_core/network_builder.py

```diff
@@ -353,6 +352,11 @@ def _create_cells_and_feeds(self):
         External inputs are not targets.
         """
         params = self.net.params
+        # Re-create external feed param dictionaries
```

Comment:
should we add an "XXX" in the comment somewhere? To indicate that this is a hack (right?) and we should get rid of at some point

Reply:
Hmm, I'm not sure in this scenario what delineates a hack from a long-term bug fix. We plan to refactor the params and feed modules wrt how feed objects and their respective parameters are passed to `Network` and `NetworkBuilder`. Along the way we will end up simplifying the code by e.g. removing the need for calling `create_pext()` here. That said, I'm happy to update the comment if you think it'd be clearer.

Reply:
yeah, I agree it might go away but just leaving an XXX tells future contributors that it's a hack that needs to fixed.